mq/esworker_sqs.py

```diff
@@ -11,7 +11,7 @@
 # kombu's support for SQS is buggy
 # so this version uses boto
 # to read an SQS queue and put events into elastic search
-# in the same manner as esworker.py
+# in the same manner as esworker-eventtask.py
```

Comment:
esworker has been renamed to "esworker_eventtask.py", not "esworker-eventtask.py"

Reply:
yeah, typo, will fix